src/task/methods/ambros_m_judges_third_place/script.py

```diff
@@ -0,0 +1,527 @@
+# This script is based on an IPython notebook:
+# https://github.com/Ambros-M/Single-Cell-Perturbations-2023/blob/main/notebooks/scp-26-py-boost-recommender-system-and-et.ipynb
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from colorama import Fore, Back, Style
+from numpy.polynomial import Polynomial
+from scipy.stats import norm, skew, kurtosis
+import scipy
+from sklearn.linear_model import Ridge
+from sklearn.decomposition import PCA
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.neighbors import KNeighborsRegressor
+from sklearn.ensemble import ExtraTreesRegressor
+from itertools import combinations
+import cupy as cp
+from py_boost import GradientBoosting
+import warnings
+import zipfile
+
+warnings.simplefilter('ignore', FutureWarning)
+np.set_printoptions(linewidth=195, edgeitems=3)
+pd.set_option("min_rows", 6)
+
+## Viash start
+
+par = dict(
+    de_train_path = "/home/jovyan/git-repos/task-dge-perturbation-prediction/resources/neurips-2023-kaggle/de_train.parquet",
+    train_obs_zip_path = "/home/jovyan/git-repos/task-dge-perturbation-prediction/resources/neurips-2023-kaggle/train_obs.csv.zip",
+    id_map_path = "/home/jovyan/git-repos/task-dge-perturbation-prediction/resources/neurips-2023-kaggle/id_map.csv",
+    output_path = "output.parquet",
+)
```

Comment:
Turn this into a proper Viash placeholder. Viash will detect the `## VIASH START` -- `## VIASH END` tags and replace it with an argument parser when the component is transformed into an executable or a nextflow module.

```suggestion
## VIASH START
par = dict(
    de_train_h5ad = "resources/neurips-2023-data/de_train.h5ad",
    id_map = "resources/neurips-2023-data/id_map.csv",
    output_path = "output.parquet",
)
## VIASH END
```